Blog post about Servo on conferences (#101)
servo/servo.org: _posts/2023-06-19-conference-news-around-servo-project.md

---
layout:     post
tags:       blog
title:      Conference news around Servo project
date:       2023-06-19
summary:    After the project reactivation early this year, Servo project is getting some presence in different events.
categories:
---

> In May 2023, Martin Robinson from the Servo project team had the chance to participate in [**RustNL 2023**](https://2023.rustnl.org/) conference in Amsterdam. There he gave a [lighting talk about Servo project](https://www.youtube.com/watch?v=IdHvHoAO5oo) and the current plans. In addition, he participated in some follow up discussions about Servo and how some UI frameworks in the Rust ecosystem are exploring the possibility to use it.

Reviewer: ```suggestion
In May of 2023, Martin Robinson from the Servo project team had the chance to participate in the [**RustNL 2023**](https://2023.rustnl.org/) conference in Amsterdam. There he gave a [lighting talk about the Servo project](https://www.youtube.com/watch?v=IdHvHoAO5oo) and its current plans. In addition, he participated in some follow up discussions exploring the possibility of using Servo in various Rust application frameworks.
```

Apart from that in the same event there was a [breakout session about Servo](https://github.com/Igalia/webengineshackfest/issues/16) where various topics were addressed, such as webrender, style crate, and the synchronization process for both their current and future developments. Of particular interest was the implementation of the Embedding API and the associated challenges. Lastly, there was a discussion on simplifying the contribution process to the Servo project, with a focus on making it more beginner-friendly, enabling a broader participation from interested individuals. You can check the [session notes online](https://github.com/Igalia/webengineshackfest/wiki/2023-Servo).

> End of the June, Servo is going to be present at [**Embedded Open Source Summit**](https://events.linuxfoundation.org/embedded-open-source-summit/) in Prague. If you are participating in such event, feel free to [approach us](mailto:info@servo.org) and ask any question about the project status and future plans.

Reviewer: ```suggestion
Around the end of  June, Servo will attend the [**Embedded Open Source Summit**](https://events.linuxfoundation.org/embedded-open-source-summit/) in Prague. If you are participating in this event, feel free to [contact us](mailto:info@servo.org) with any questions about the project status and our plans for the future.
```
